content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -706,7 +706,7 @@ kind: Namespace
 metadata:
   name: apps-that-need-nodes-exclusively
   annotations:
-    scheduler.alpha.kubernetes.io/defaultTolerations: '{"operator": "Exists", "effect": "NoSchedule", "key": "dedicated-node"}'
+    scheduler.alpha.kubernetes.io/defaultTolerations: '[{"operator": "Exists", "effect": "NoSchedule", "key": "dedicated-node"}]'
     scheduler.alpha.kubernetes.io/tolerationsWhitelist: '{"operator": "Exists", "effect": "NoSchedule", "key": "dedicated-node"}'
```

Comment:
should we change line 710 as well?

Reply:
correct, just tested that and fixed in latest commit